src/providers/sms/bulutfon.js

```diff
@@ -0,0 +1,40 @@
+/* @flow */
+import Request from 'request-promise-native'
+// Types
+import type {SmsRequestType} from '../../models/notification-request'
+
+export default class BulutfonProvider {
+  id: string
+  config: Object
+
+  constructor (config: Object) {
+    this.id = 'sms-bulutfon-provider'
+    this.config = {
+      defaultTitle: config.defaultTitle,
+      access_token: config.accessToken
+    }
+  }
+
+  async send (request: SmsRequestType): Promise<string> {
+    const { defaultTitle: title = request.from, access_token } = this.config
+
+    let receivers = request.to
+    if (Array.isArray(receivers)) {
+      receivers = receivers.join(',')
+    }
+
+    try {
+      const result = await Request
+        .post('https://api.bulutfon.com/messages')
```

Comment:
It seems that I can't create an account to test bulutfon's api :/
From their api doc, it seems that they send the parameters in the query string instead of the form data. Can you confirm that both are working?